modules/ROOT/pages/tuning-domains.adoc

```diff
@@ -0,0 +1,17 @@
+= Domains
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Besides the listed benefits on the Shared Resources MuleSoft documentation, domains are also a performance choice when deploying multiple services on the same on-premises Mule runtime engine (Mule). By having a centralized point for all the shared resources, domains help the class-loading process, therefore metaspace memory usage, to be more efficient, especially because dependencies of the domain (those declared in the `pom.xml` file) are also shared in the domain apps.
```

Comment:
```suggestion
Among its many other shared-resource benefits, domain use greatly enhances processing performance when you deploy multiple services on a single on-premises Mule runtime engine (Mule). By providing centralized resource sharing, domains help make the class-loading process (and, therefore, metaspace memory usage) more efficient, especially because the domain dependencies declared in the `pom.xml` file are also shared by the domain apps.
```